wire-ios-data-model/Source/ManagedObjectContext/Migration/2.106-2.107/MappingModel_2.106-2.107.xcmappingmodel/xcmapping.xml

```diff
@@ -10,7 +10,9 @@
             <plist version="1.0">
                 <dict>
                     <key>NSPersistenceFrameworkVersion</key>
-                    <integer>1251</integer>
+                    <integer>1338</integer>
```

Comment:
Is that ok ? Do we brek some compatibility ?

Reply:
I don't think it breaks something, because the migration is only executed once per client and thus it's different than the `.xcdatamodel` versions.